src/devices/bus/megadrive/cart/segach.cpp

```diff
@@ -100,11 +112,35 @@ megadrive_segach_us_device::megadrive_segach_us_device(const machine_config &mco
 	: device_t(mconfig, MEGADRIVE_SEGACH_US, tag, owner, clock)
 	, device_megadrive_cart_interface( mconfig, *this )
 	, device_memory_interface( mconfig, *this )
+	, m_packet_timer(*this, "packet_timer")
 	, m_rom(*this, "rom")
-	, m_space_tcu_config("tcu_io", ENDIANNESS_BIG, 16, 9, 0, address_map_constructor(FUNC(megadrive_segach_us_device::tcu_map), this))
+	, m_space_tcu_config("tcu_io", ENDIANNESS_BIG, 8, 9, 0, address_map_constructor(FUNC(megadrive_segach_us_device::tcu_map), this))
+	, m_sram_view(*this, "sram_view")
+	, m_game_view(*this, "game_view")
+	, m_game_sram_view(*this, "game_sram_view")
 {
 }
 
+void megadrive_segach_us_device::device_add_mconfig(machine_config &config)
+{
+	// each broadcast frame has 10 packets and 288 bytes per packet
+	// assuming 12Mbps download rate
+	TIMER(config, "packet_timer", 0).configure_periodic(FUNC(megadrive_segach_us_device::send_packets),
+			attotime::from_hz((12*128072) / 2880));
+
+	QUICKLOAD(config, "packet_stream", "img").set_load_callback(FUNC(megadrive_segach_us_device::quickload_cb));
+}
+
+static INPUT_PORTS_START( megadrive_segach_us )
+	PORT_START("BUTTON")
+	PORT_BIT( 0x01, IP_ACTIVE_LOW, IPT_BUTTON1 ) PORT_NAME("Menu") PORT_CHANGED_MEMBER(DEVICE_SELF, FUNC(megadrive_segach_us_device::menu_pressed), 0)
```

Comment:
This probably needs to be `IPT_OTHER` (so that it doesn't map to anything by default)